go.mod

```diff
@@ -2,10 +2,6 @@ module github.com/textileio/go-threads
 
 go 1.14
 
-replace github.com/ipfs/go-datastore v0.4.4 => github.com/textileio/go-datastore v0.4.5-0.20200728205504-ffeb3591b248
```

Comment:
No longer need these... `net` still needs `ipfs/go-datastore` since it interacts with things like `Blockstore`, which depend on `ipfs/go-datastore`, but `db` is using `textileio/go-datastore`.